packages/manager/apps/cloud/client/app/dbaas/logs/detail/index/add/logs-index-add.controller.js

```diff
@@ -28,21 +28,62 @@ class LogsIndexAddModalCtrl {
       this.populateIndex();
     } else {
       this.clearIndex();
+      this.accountDetails.load().then(() => {
+        this.ovhSubsidiary = this.accountDetails.data.me.ovhSubsidiary;
+        this.$q.all([this.mainOffer.load(), this.catalog.load()]).then(() => {
+          const selectedCatalog = this.catalog.data.plans.find(
+            (plan) => plan.planCode === this.mainOffer.data.planCode,
+          );
+          const selectedFamily = selectedCatalog.addonsFamily.find(
+            (addon) => addon.family === this.LogsConstants.ADD_ON_FAMILY.NEW,
+          );
+          const indexShard = selectedFamily.addons.find(
+            (add) =>
+              add.plan.planCode ===
+              this.LogsConstants.CONSUMPTION_REFERENCE.NB_SHARD,
+          );
+          const indexVolume = selectedFamily.addons.find(
+            (add) =>
+              add.plan.planCode ===
+              this.LogsConstants.CONSUMPTION_REFERENCE.INDEXED_DOCUMENTS,
+          );
+          this.indexPrice.shard.price =
+            indexShard.plan.details.pricings.default[0].price.text;
```

Comment:
We should avoid taking the first pricing by default, this has caused some issues previously. maybe we can use the `capacities` instead ?

Reply:
Not sure to understand, `capacities` give me installation, upgrade or setup values. 
We don't have any particular fees regarding theses aspects. Displayed price is self sufficient, no hidden cost.

Reply:
She is right though, for two reasons:
- if it changes, your code will break (don't tell me it won't change, you guys always say that right beofre changing everything 😄 )
- your code doesn't state its intention (it's the first price, but what's the first price ? It's the price for the installation ? For the upgrade ?)

Reply:
My wonder is what if we suddenly separate installation from upgrade. Let's say installation is free but it's the first price so we'll display wrong price to the customer. 
We should take the price according to what we charge customer (installation, renew, upgrade ...) not the first price because currently there is only one

Reply:
It's a strict **consumption** billing add-on, they are not any setup, upgrade, install fees, the final price depends of the amount of resource consumed for the period. Is that more clear ?

Reply:
Got it, index[0] has been replaced by capacity check, in case billing team decide to introduce new ones on such kind of add on.